### ISSUE TITLE
Reduce errors in table output

### ISSUE THREAD

**Issue by jfheins** [sensslen/nuget-license#61]
First a thank you for the nice tool.
I used it on our project and ran this command:
> dotnet nuget-license -i All-Projects.sln --include-transitive -o table > licenses.txt 

This gives me a huge file with a lot of errors about project references. There are a few gaps in the rows in the screenshot. 
![image](https://github.com/sensslen/nuget-license/assets/5293502/6f1f9286-08ba-4c06-9ee2-8d6eb0539fd0)
The gaps have some useless error on the very right:
> Validation for licenses of type File not yet supported | /Users/xx/Repos/MyProject/Backend/Shared/Core.csproj

Right now I use awk to remove the errors from the output and post-process the table:
> awk '{ print $5 $6 }' FS='|' lic-be.txt | tail -n +3 | sort | uniq > licenses/backend.txt

But it would ne nice if there was an option that either ignores all the errors or excludes all my csproj files.

**Comment by jfheins** [comment]
Maybe I can shed some more light on the use case. My idea was to analyze the project and dump all relevant license info. First attempt would be a text file that contains all license URLs. (Strictly speaking I guess it has to be license file contents)
Then the pipeline would detect changes, commit them and changes in the license folder require a special approver group.

Effectively stopping people from adding licenses without hesitation.
The license approver group would need to check if the new license is acceptable.

So in short, I don't really care about the current license terms. I want to catch the case where the license changes (e.g. after a nuget update). For me, the first step is always change control. Then in the second step one can verify that the licenses are acceptable.

**Comment by sensslen** [comment]
Sorry to disappoint you here. With the licenseUrl people have used various url's that point to complex page involving javascript and that change content between visits (not the license necessarily but the html). 

As I said, I don't really see a good way of including your request in nuget-license. Basically what you are suggesting is a different tool that uses more or less the same infrastructure nuget-license does.

Feel free to work on creating a seperate tool based on nuget-license that does not validate licenses but just download the license (be it a file, url or SPDX identifier).
